src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/ReadResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,56 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Read" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify existing resources can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
```

Comment:
It would be good to have the expected response body here:

```json
[
  {
        "id": "{id}"
        "codeValue": "Jury duty",
        "description": "Jury duty",
        "effectiveBeginDate": "2024-05-14",
        "effectiveEndDate": "2024-05-14",
        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
        "shortDescription": "Jury duty",
        "_etag": "5250182869515346067",
        "_lastModifiedDate": "2024-04-25T23:38:28.7958163Z"
    }
]
```